files/btrust.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=Btrust
+After=bluetooth.service
```

Comment:
If this tool uses `pulseaudio`, it should be listed as dependency.

Reply:
The role installs pulseaudio